tic_tac_toe.rb

```diff
@@ -1,13 +1,13 @@
 require './lib/game_session'
 require './lib/player'
 require './lib/board'
-require './lib/rules'
+require './lib/outcome'
 require './lib/display'
 
 player1 = Player.new(' X')
 player2 = Player.new(' O')
 display1 = Display.new
-rules = Rules.new
+outcome = Outcome.new
 
-game_session = GameSession.new(player1, player2, display1, rules)
+game_session = GameSession.new(player1, player2, display1, outcome)
```

Comment:
Why is it called `display1` (suggests a sequence of displays)? I would just call `display`.

Reply:
I called it 'display1' because when I initially had it as 'display' the word was highlighted and I thought it was a special key word. But I just changed everything to display and nothing broke. Done!